Assets/Scripts/Game/UserInterfaceWindows/DaggerfallUnitySetupGameWizard.cs

```diff
@@ -423,13 +423,30 @@ HorizontalSlider AddSlider(float x, string key, int selected, params string[] ch
                 optionsPanel
             );
             slider.ToolTip = defaultToolTip;
-            slider.ToolTipText = GetText("weaponSwingModeInfo");
+            slider.ToolTipText = GetText(toolTipKey);
 
             optionPos += optionSpacing;
 
             return slider;
         }
 
+        HorizontalSlider AddSlider(float x, string key, int selected, string toolTipKey, float minValue, float maxValue)
+        {
+            int min = Mathf.RoundToInt(minValue * 10);
+            int max = Mathf.RoundToInt(maxValue * 10);
+
+            int numValues = max - min + 1;
+
+            string[] choices = new string[numValues];
+
+            --min;
+
+            for (int i = 0; i < numValues; ++i)
```

Comment:
Can you explain this to me? There should be no need to add multiple choices here, the slider control can do 0.0 to 0.9 in 0.1 increments if that's what you're aiming for?

Reply:
Just a bit of shoe-horning. Has been reworked to instantiate proper float slider.